docs/source/contents/4_draft_mapping.rst

```diff
@@ -1662,6 +1662,459 @@ opaque:hostItem
+                    <text>County: Shelby County</text>
+                    <text>State: Tennessee</text>
+                </physicalAddress>
+            </holding>
+        </holdingExternal>
+    </location>
+
+.. code-block:: turtle
+
+    @prefix relators: <http://id.loc.gov/vocabulary/relators/> .
+    @prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+
+    <https://example.org/objects/1>
+        relators:rps "University of Memphis. Special Collections" ;
+        skos:note "Shelf locator: Manuscript Number 5" .
+
+physicalLocation with @displayLabel="Address"
+---------------------------------------------
+
+Use Case
+^^^^^^^^
+
+Some of items with the physicalLocation of Pi Beta Phi Fraternity also have a physicalLocation subelement with the displayLabel attribute of Address.
+
+Justification
+^^^^^^^^^^^^^
+
+Similar to the holdingExternal, we will opt drop this information to maintain simplicity of our data from a technical standpoint.
+
+Xpath
+^^^^^
+
+:code:`mods:location/mods:physicalLocation[@displayLabel="Address"]`
+
+Decision
+^^^^^^^^
+Drop this.
+
+`Example record from volvoices:2199 <https://digital.lib.utk.edu/collections/islandora/object/volvoices:2199/datastream/MODS/view>`_
+
+.. code-block:: xml
+
+    <location>
+        <physicalLocation>Pi Beta Phi Fraternity</physicalLocation>
+        <physicalLocation displayLabel="Address">1154 Town and Country Commons Drive, Town and Country, Missouri 63017</physicalLocation>
+        <shelfLocator>Box 36, Folder 14</shelfLocator>
+    </location>
+
+.. code-block:: turtle
+
+    @prefix relators: <http://id.loc.gov/vocabulary/relators/> .
+    @prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+
+    <https://example.org/objects/1>
+        relators:rps "Pi Beta Phi Fraternity" ;
+        skos:note "Shelf locator: Box 36, Folder 14" .
+
+physicalLocation with @displayLabel="Collection"
+------------------------------------------------
+
+Use Case
+^^^^^^^^
+In a some collections for Arrowmont, we will find items having a physicalLocation subelement with the displayLabel attribute of Collection with text containing "Archives Collection". We also have extra physicalLocation subelements with displayLabel attributes of Detailed Location, City and State.
+
+Justification
+^^^^^^^^^^^^^
+Because these records do not already have a dbo:collection predicate, we will transcribe the string literal to dbo:collection for :code:`mods:location/mods:physicalLocation[@displayLabel="Collection"]`. No other data here needs to be retained and will be dropped.
+
+Xpath
+^^^^^
+
+:code:`mods:location[mods:physicalLocation[@displayLabel="Collection" and text()[contains(.,"Archives Collection")]]]`
+:code:`mods:location[mods:physicalLocation[@displayLabel="Repository"]]`
+:code:`mods:location[mods:physicalLocation[@displayLabel="Detailed Location"]]`
+:code:`mods:location[mods:physicalLocation[@displayLabel="City"]]`
+:code:`mods:location[mods:physicalLocation[@displayLabel="State"]]`
```

Comment:
The way that it's written here makes it read as if the discussion refers to what to do with the entire location node when it matches any of the conditions, but that's not the case from our conversation.  Switch to this:

```
:code:`mods:location/mods:physicalLocation[@displayLabel="Collection" and text()[contains(.,"Archives Collection")]]`
:code:`mods:location/mods:physicalLocation[@displayLabel="Repository"]`
:code:`mods:location/mods:physicalLocation[@displayLabel="Detailed Location"]`
:code:`mods:location/mods:physicalLocation[@displayLabel="City"]`
:code:`mods:location/mods:physicalLocation[@displayLabel="State"]`
```

Reply:
Thanks @markpbaggett , it's pushed now, see: https://github.com/UTKcataloging/mods_to_rdf/pull/37/commits/2bd279db531244cc64ab3b58b82ddec46f00ae24

Reply:
I'm so confused. Did you just push? I'm trying to figure out why the Actions bot just ran and why I didn't see this commit. It definitely seems like @mlhale7 could see this and maybe so could @CanOfBees, but I kept refreshing to see why I couldn't see it and obviously Actions wasn't seeing it either.